scamp/scamp-app.c

```diff
@@ -131,7 +131,17 @@ void set_cpu_info(uint virt_mask, uint state, uint app_id)
 //! \return The bit index of the least significant set bit
```

Comment:
GNU documentation for Built-in Function: int __builtin_ffs (int x)

> Returns one plus the index of the least significant 1-bit of x, or if x is zero, returns zero.

This is __not__ consistent with the comment in our function header and implementation.

Reply:
Yes, looks like there would have to be a check for 0 and a -1 in there to use it; maybe we just need to use the function as it is instead...

Reply:
I have reverted this back to the original code without the builtin function.

Reply:
Should it have been something more like this? https://stackoverflow.com/a/42030874/301832

Reply:
This is really "count trailing zeros" I think (but I could be wrong).

Reply:
In any case, the function isn't used in a critical part of the code I don't think...